File: Find_middle_element_in_a_linked_list/code.java
class Solution
{
    int getMiddle(Node head)
    {
         // Your code here.
         Node temp=head;
         int count=0;
         while(temp!=null)
         {
             count++;
             temp=temp.next;
         }
         int index=(count/2)+1;
         Node curr=head;
         while(--index!=0)
         {
             curr=curr.next;
         }
         
         return curr.data;
    }
}

File: Column_Name_from_a_Given_Column_Number/code.cpp
class Solution{
    public:
    string colName (long long int n)
    {
        // your code 
        string res = "";
        while(n>0){
            n--;
            char temp = 'A'+n%26;
            n=n/26;
            res+=temp;
        }
        reverse(res.begin(),res.end());
        return res;
    }
};